pkg/contracts/ton/gateway_deploy.go

```diff
@@ -0,0 +1,74 @@
+package ton
+
+import (
+	_ "embed"
+	"encoding/json"
+
+	eth "github.com/ethereum/go-ethereum/common"
+	"github.com/pkg/errors"
+	"github.com/tonkeeper/tongo/boc"
+	"github.com/tonkeeper/tongo/tlb"
+	"github.com/tonkeeper/tongo/ton"
+)
+
+//go:embed gateway.compiled.json
+var gatewayCode []byte
+
+// GatewayCode returns Gateway's code as cell
+func GatewayCode() *boc.Cell {
+	c, err := getGatewayCode()
+	if err != nil {
+		panic(err)
```

Comment:
Do we need a panic here and next function?

Reply:
yes, that's indented. Just like `index out of range` panic for a slice. 

The idea is that this code might only panic if something unexpectedly goes wrong with the build.

If we can't get gw's code, then it makes no sense to run the binary, thus I removed `error` from return values.